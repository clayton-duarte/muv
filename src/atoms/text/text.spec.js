import React from 'react';

import Text from '.';

it('should render a text', () => {
  const wrapper = mount(
    <Text />,
  );
  expect(wrapper).toMatchSnapshot();
});

it('should change its style if strong', () => {
  const wrapper = mount(
    <Text strong />,
  );
  expect(wrapper).toMatchSnapshot();
});

it('should change its style on warn', () => {
  const wrapper = mount(
    <Text warn />,
  );
  expect(wrapper).toMatchSnapshot();
});

it('should change its style on success', () => {
  const wrapper = mount(
    <Text success />,
  );
  expect(wrapper).toMatchSnapshot();
});

it('should change its style on primary', () => {
  const wrapper = mount(
    <Text primary />,
  );
  expect(wrapper).toMatchSnapshot();
});

it('should change its style on secondary', () => {
  const wrapper = mount(
    <Text secondary />,
  );
  expect(wrapper).toMatchSnapshot();
});

it('should change its content concerning the type', () => {
  const h1 = mount(
    <Text type="h1" />,
  );
  expect(h1).toMatchSnapshot();
  const h2 = mount(
    <Text type="h2" />,
  );
  expect(h2).toMatchSnapshot();
  const h3 = mount(
    <Text type="h3" />,
  );
  expect(h3).toMatchSnapshot();
  const h4 = mount(
    <Text type="h4" />,
  );
  expect(h4).toMatchSnapshot();
  const h5 = mount(
    <Text type="h5" />,
  );
  expect(h5).toMatchSnapshot();
  const h6 = mount(
    <Text type="h6" />,
  );
  expect(h6).toMatchSnapshot();
});
