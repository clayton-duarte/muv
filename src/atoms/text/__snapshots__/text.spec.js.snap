// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should change its content concerning the type 1`] = `
<Text
  align="left"
  label={false}
  strong={false}
  theme={
    Object {
      "color": Object {
        "black": "#484848",
        "gray": "#f0f0f0",
        "overlay": "rgba(0, 0, 0, .15)",
        "primary": "#0082c8",
        "secondary": "#00baf2",
        "success": "#33bb33",
        "warn": "#dd3333",
        "white": "#fefefe",
      },
      "shape": Object {
        "base": "1rem",
        "border": ".125rem solid rgba(0, 0, 0, .15)",
        "margin": ".25rem .125rem",
        "padding": ".825rem 1.2rem",
        "radius": ".25rem",
        "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
      },
      "text": Object {
        "color": "#484848",
        "size": "1rem",
      },
      "transition": Object {
        "time": ".2s ease",
      },
    }
  }
  transform="normal"
  type="h1"
>
  <styled.h1
    align="left"
    label={false}
    strong={true}
    theme={
      Object {
        "color": Object {
          "black": "#484848",
          "gray": "#f0f0f0",
          "overlay": "rgba(0, 0, 0, .15)",
          "primary": "#0082c8",
          "secondary": "#00baf2",
          "success": "#33bb33",
          "warn": "#dd3333",
          "white": "#fefefe",
        },
        "shape": Object {
          "base": "1rem",
          "border": ".125rem solid rgba(0, 0, 0, .15)",
          "margin": ".25rem .125rem",
          "padding": ".825rem 1.2rem",
          "radius": ".25rem",
          "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
        },
        "text": Object {
          "color": "#484848",
          "size": "1rem",
        },
        "transition": Object {
          "time": ".2s ease",
        },
      }
    }
    transform="normal"
  >
    <h1
      className="sc-htpNat clpkGo"
      label={false}
      transform="normal"
    >
      default
    </h1>
  </styled.h1>
</Text>
`;

exports[`should change its content concerning the type 2`] = `
<Text
  align="left"
  label={false}
  strong={false}
  theme={
    Object {
      "color": Object {
        "black": "#484848",
        "gray": "#f0f0f0",
        "overlay": "rgba(0, 0, 0, .15)",
        "primary": "#0082c8",
        "secondary": "#00baf2",
        "success": "#33bb33",
        "warn": "#dd3333",
        "white": "#fefefe",
      },
      "shape": Object {
        "base": "1rem",
        "border": ".125rem solid rgba(0, 0, 0, .15)",
        "margin": ".25rem .125rem",
        "padding": ".825rem 1.2rem",
        "radius": ".25rem",
        "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
      },
      "text": Object {
        "color": "#484848",
        "size": "1rem",
      },
      "transition": Object {
        "time": ".2s ease",
      },
    }
  }
  transform="normal"
  type="h2"
>
  <styled.h2
    align="left"
    label={false}
    strong={true}
    theme={
      Object {
        "color": Object {
          "black": "#484848",
          "gray": "#f0f0f0",
          "overlay": "rgba(0, 0, 0, .15)",
          "primary": "#0082c8",
          "secondary": "#00baf2",
          "success": "#33bb33",
          "warn": "#dd3333",
          "white": "#fefefe",
        },
        "shape": Object {
          "base": "1rem",
          "border": ".125rem solid rgba(0, 0, 0, .15)",
          "margin": ".25rem .125rem",
          "padding": ".825rem 1.2rem",
          "radius": ".25rem",
          "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
        },
        "text": Object {
          "color": "#484848",
          "size": "1rem",
        },
        "transition": Object {
          "time": ".2s ease",
        },
      }
    }
    transform="normal"
  >
    <h2
      className="sc-bxivhb imVoGD"
      label={false}
      transform="normal"
    >
      default
    </h2>
  </styled.h2>
</Text>
`;

exports[`should change its content concerning the type 3`] = `
<Text
  align="left"
  label={false}
  strong={false}
  theme={
    Object {
      "color": Object {
        "black": "#484848",
        "gray": "#f0f0f0",
        "overlay": "rgba(0, 0, 0, .15)",
        "primary": "#0082c8",
        "secondary": "#00baf2",
        "success": "#33bb33",
        "warn": "#dd3333",
        "white": "#fefefe",
      },
      "shape": Object {
        "base": "1rem",
        "border": ".125rem solid rgba(0, 0, 0, .15)",
        "margin": ".25rem .125rem",
        "padding": ".825rem 1.2rem",
        "radius": ".25rem",
        "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
      },
      "text": Object {
        "color": "#484848",
        "size": "1rem",
      },
      "transition": Object {
        "time": ".2s ease",
      },
    }
  }
  transform="normal"
  type="h3"
>
  <styled.h3
    align="left"
    label={false}
    strong={true}
    theme={
      Object {
        "color": Object {
          "black": "#484848",
          "gray": "#f0f0f0",
          "overlay": "rgba(0, 0, 0, .15)",
          "primary": "#0082c8",
          "secondary": "#00baf2",
          "success": "#33bb33",
          "warn": "#dd3333",
          "white": "#fefefe",
        },
        "shape": Object {
          "base": "1rem",
          "border": ".125rem solid rgba(0, 0, 0, .15)",
          "margin": ".25rem .125rem",
          "padding": ".825rem 1.2rem",
          "radius": ".25rem",
          "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
        },
        "text": Object {
          "color": "#484848",
          "size": "1rem",
        },
        "transition": Object {
          "time": ".2s ease",
        },
      }
    }
    transform="normal"
  >
    <h3
      className="sc-ifAKCX kMguy"
      label={false}
      transform="normal"
    >
      default
    </h3>
  </styled.h3>
</Text>
`;

exports[`should change its content concerning the type 4`] = `
<Text
  align="left"
  label={false}
  strong={false}
  theme={
    Object {
      "color": Object {
        "black": "#484848",
        "gray": "#f0f0f0",
        "overlay": "rgba(0, 0, 0, .15)",
        "primary": "#0082c8",
        "secondary": "#00baf2",
        "success": "#33bb33",
        "warn": "#dd3333",
        "white": "#fefefe",
      },
      "shape": Object {
        "base": "1rem",
        "border": ".125rem solid rgba(0, 0, 0, .15)",
        "margin": ".25rem .125rem",
        "padding": ".825rem 1.2rem",
        "radius": ".25rem",
        "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
      },
      "text": Object {
        "color": "#484848",
        "size": "1rem",
      },
      "transition": Object {
        "time": ".2s ease",
      },
    }
  }
  transform="normal"
  type="h4"
>
  <styled.h4
    align="left"
    label={false}
    strong={true}
    theme={
      Object {
        "color": Object {
          "black": "#484848",
          "gray": "#f0f0f0",
          "overlay": "rgba(0, 0, 0, .15)",
          "primary": "#0082c8",
          "secondary": "#00baf2",
          "success": "#33bb33",
          "warn": "#dd3333",
          "white": "#fefefe",
        },
        "shape": Object {
          "base": "1rem",
          "border": ".125rem solid rgba(0, 0, 0, .15)",
          "margin": ".25rem .125rem",
          "padding": ".825rem 1.2rem",
          "radius": ".25rem",
          "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
        },
        "text": Object {
          "color": "#484848",
          "size": "1rem",
        },
        "transition": Object {
          "time": ".2s ease",
        },
      }
    }
    transform="normal"
  >
    <h4
      className="sc-EHOje irSNBG"
      label={false}
      transform="normal"
    >
      default
    </h4>
  </styled.h4>
</Text>
`;

exports[`should change its content concerning the type 5`] = `
<Text
  align="left"
  label={false}
  strong={false}
  theme={
    Object {
      "color": Object {
        "black": "#484848",
        "gray": "#f0f0f0",
        "overlay": "rgba(0, 0, 0, .15)",
        "primary": "#0082c8",
        "secondary": "#00baf2",
        "success": "#33bb33",
        "warn": "#dd3333",
        "white": "#fefefe",
      },
      "shape": Object {
        "base": "1rem",
        "border": ".125rem solid rgba(0, 0, 0, .15)",
        "margin": ".25rem .125rem",
        "padding": ".825rem 1.2rem",
        "radius": ".25rem",
        "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
      },
      "text": Object {
        "color": "#484848",
        "size": "1rem",
      },
      "transition": Object {
        "time": ".2s ease",
      },
    }
  }
  transform="normal"
  type="h5"
>
  <styled.h5
    align="left"
    label={false}
    strong={true}
    theme={
      Object {
        "color": Object {
          "black": "#484848",
          "gray": "#f0f0f0",
          "overlay": "rgba(0, 0, 0, .15)",
          "primary": "#0082c8",
          "secondary": "#00baf2",
          "success": "#33bb33",
          "warn": "#dd3333",
          "white": "#fefefe",
        },
        "shape": Object {
          "base": "1rem",
          "border": ".125rem solid rgba(0, 0, 0, .15)",
          "margin": ".25rem .125rem",
          "padding": ".825rem 1.2rem",
          "radius": ".25rem",
          "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
        },
        "text": Object {
          "color": "#484848",
          "size": "1rem",
        },
        "transition": Object {
          "time": ".2s ease",
        },
      }
    }
    transform="normal"
  >
    <h5
      className="sc-bZQynM bnaBhs"
      label={false}
      transform="normal"
    >
      default
    </h5>
  </styled.h5>
</Text>
`;

exports[`should change its content concerning the type 6`] = `
<Text
  align="left"
  label={false}
  strong={false}
  theme={
    Object {
      "color": Object {
        "black": "#484848",
        "gray": "#f0f0f0",
        "overlay": "rgba(0, 0, 0, .15)",
        "primary": "#0082c8",
        "secondary": "#00baf2",
        "success": "#33bb33",
        "warn": "#dd3333",
        "white": "#fefefe",
      },
      "shape": Object {
        "base": "1rem",
        "border": ".125rem solid rgba(0, 0, 0, .15)",
        "margin": ".25rem .125rem",
        "padding": ".825rem 1.2rem",
        "radius": ".25rem",
        "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
      },
      "text": Object {
        "color": "#484848",
        "size": "1rem",
      },
      "transition": Object {
        "time": ".2s ease",
      },
    }
  }
  transform="normal"
  type="h6"
>
  <styled.h6
    align="left"
    label={false}
    strong={true}
    theme={
      Object {
        "color": Object {
          "black": "#484848",
          "gray": "#f0f0f0",
          "overlay": "rgba(0, 0, 0, .15)",
          "primary": "#0082c8",
          "secondary": "#00baf2",
          "success": "#33bb33",
          "warn": "#dd3333",
          "white": "#fefefe",
        },
        "shape": Object {
          "base": "1rem",
          "border": ".125rem solid rgba(0, 0, 0, .15)",
          "margin": ".25rem .125rem",
          "padding": ".825rem 1.2rem",
          "radius": ".25rem",
          "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
        },
        "text": Object {
          "color": "#484848",
          "size": "1rem",
        },
        "transition": Object {
          "time": ".2s ease",
        },
      }
    }
    transform="normal"
  >
    <h6
      className="sc-gzVnrw jQTwaD"
      label={false}
      transform="normal"
    >
      default
    </h6>
  </styled.h6>
</Text>
`;

exports[`should change its style if strong 1`] = `
<Text
  align="left"
  label={false}
  strong={true}
  theme={
    Object {
      "color": Object {
        "black": "#484848",
        "gray": "#f0f0f0",
        "overlay": "rgba(0, 0, 0, .15)",
        "primary": "#0082c8",
        "secondary": "#00baf2",
        "success": "#33bb33",
        "warn": "#dd3333",
        "white": "#fefefe",
      },
      "shape": Object {
        "base": "1rem",
        "border": ".125rem solid rgba(0, 0, 0, .15)",
        "margin": ".25rem .125rem",
        "padding": ".825rem 1.2rem",
        "radius": ".25rem",
        "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
      },
      "text": Object {
        "color": "#484848",
        "size": "1rem",
      },
      "transition": Object {
        "time": ".2s ease",
      },
    }
  }
  transform="normal"
  type=""
>
  <styled.p
    align="left"
    label={false}
    strong={true}
    theme={
      Object {
        "color": Object {
          "black": "#484848",
          "gray": "#f0f0f0",
          "overlay": "rgba(0, 0, 0, .15)",
          "primary": "#0082c8",
          "secondary": "#00baf2",
          "success": "#33bb33",
          "warn": "#dd3333",
          "white": "#fefefe",
        },
        "shape": Object {
          "base": "1rem",
          "border": ".125rem solid rgba(0, 0, 0, .15)",
          "margin": ".25rem .125rem",
          "padding": ".825rem 1.2rem",
          "radius": ".25rem",
          "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
        },
        "text": Object {
          "color": "#484848",
          "size": "1rem",
        },
        "transition": Object {
          "time": ".2s ease",
        },
      }
    }
    transform="normal"
  >
    <p
      className="sc-bdVaJa hNDhZz"
      label={false}
      transform="normal"
    >
      default
    </p>
  </styled.p>
</Text>
`;

exports[`should change its style on primary 1`] = `
<Text
  align="left"
  label={false}
  primary={true}
  strong={false}
  theme={
    Object {
      "color": Object {
        "black": "#484848",
        "gray": "#f0f0f0",
        "overlay": "rgba(0, 0, 0, .15)",
        "primary": "#0082c8",
        "secondary": "#00baf2",
        "success": "#33bb33",
        "warn": "#dd3333",
        "white": "#fefefe",
      },
      "shape": Object {
        "base": "1rem",
        "border": ".125rem solid rgba(0, 0, 0, .15)",
        "margin": ".25rem .125rem",
        "padding": ".825rem 1.2rem",
        "radius": ".25rem",
        "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
      },
      "text": Object {
        "color": "#484848",
        "size": "1rem",
      },
      "transition": Object {
        "time": ".2s ease",
      },
    }
  }
  transform="normal"
  type=""
>
  <styled.p
    align="left"
    label={false}
    primary={true}
    strong={false}
    theme={
      Object {
        "color": Object {
          "black": "#484848",
          "gray": "#f0f0f0",
          "overlay": "rgba(0, 0, 0, .15)",
          "primary": "#0082c8",
          "secondary": "#00baf2",
          "success": "#33bb33",
          "warn": "#dd3333",
          "white": "#fefefe",
        },
        "shape": Object {
          "base": "1rem",
          "border": ".125rem solid rgba(0, 0, 0, .15)",
          "margin": ".25rem .125rem",
          "padding": ".825rem 1.2rem",
          "radius": ".25rem",
          "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
        },
        "text": Object {
          "color": "#484848",
          "size": "1rem",
        },
        "transition": Object {
          "time": ".2s ease",
        },
      }
    }
    transform="normal"
  >
    <p
      className="sc-bdVaJa MKBMS"
      label={false}
      transform="normal"
    >
      default
    </p>
  </styled.p>
</Text>
`;

exports[`should change its style on secondary 1`] = `
<Text
  align="left"
  label={false}
  secondary={true}
  strong={false}
  theme={
    Object {
      "color": Object {
        "black": "#484848",
        "gray": "#f0f0f0",
        "overlay": "rgba(0, 0, 0, .15)",
        "primary": "#0082c8",
        "secondary": "#00baf2",
        "success": "#33bb33",
        "warn": "#dd3333",
        "white": "#fefefe",
      },
      "shape": Object {
        "base": "1rem",
        "border": ".125rem solid rgba(0, 0, 0, .15)",
        "margin": ".25rem .125rem",
        "padding": ".825rem 1.2rem",
        "radius": ".25rem",
        "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
      },
      "text": Object {
        "color": "#484848",
        "size": "1rem",
      },
      "transition": Object {
        "time": ".2s ease",
      },
    }
  }
  transform="normal"
  type=""
>
  <styled.p
    align="left"
    label={false}
    secondary={true}
    strong={false}
    theme={
      Object {
        "color": Object {
          "black": "#484848",
          "gray": "#f0f0f0",
          "overlay": "rgba(0, 0, 0, .15)",
          "primary": "#0082c8",
          "secondary": "#00baf2",
          "success": "#33bb33",
          "warn": "#dd3333",
          "white": "#fefefe",
        },
        "shape": Object {
          "base": "1rem",
          "border": ".125rem solid rgba(0, 0, 0, .15)",
          "margin": ".25rem .125rem",
          "padding": ".825rem 1.2rem",
          "radius": ".25rem",
          "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
        },
        "text": Object {
          "color": "#484848",
          "size": "1rem",
        },
        "transition": Object {
          "time": ".2s ease",
        },
      }
    }
    transform="normal"
  >
    <p
      className="sc-bdVaJa kMYrQE"
      label={false}
      transform="normal"
    >
      default
    </p>
  </styled.p>
</Text>
`;

exports[`should change its style on success 1`] = `
<Text
  align="left"
  label={false}
  strong={false}
  success={true}
  theme={
    Object {
      "color": Object {
        "black": "#484848",
        "gray": "#f0f0f0",
        "overlay": "rgba(0, 0, 0, .15)",
        "primary": "#0082c8",
        "secondary": "#00baf2",
        "success": "#33bb33",
        "warn": "#dd3333",
        "white": "#fefefe",
      },
      "shape": Object {
        "base": "1rem",
        "border": ".125rem solid rgba(0, 0, 0, .15)",
        "margin": ".25rem .125rem",
        "padding": ".825rem 1.2rem",
        "radius": ".25rem",
        "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
      },
      "text": Object {
        "color": "#484848",
        "size": "1rem",
      },
      "transition": Object {
        "time": ".2s ease",
      },
    }
  }
  transform="normal"
  type=""
>
  <styled.p
    align="left"
    label={false}
    strong={false}
    success={true}
    theme={
      Object {
        "color": Object {
          "black": "#484848",
          "gray": "#f0f0f0",
          "overlay": "rgba(0, 0, 0, .15)",
          "primary": "#0082c8",
          "secondary": "#00baf2",
          "success": "#33bb33",
          "warn": "#dd3333",
          "white": "#fefefe",
        },
        "shape": Object {
          "base": "1rem",
          "border": ".125rem solid rgba(0, 0, 0, .15)",
          "margin": ".25rem .125rem",
          "padding": ".825rem 1.2rem",
          "radius": ".25rem",
          "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
        },
        "text": Object {
          "color": "#484848",
          "size": "1rem",
        },
        "transition": Object {
          "time": ".2s ease",
        },
      }
    }
    transform="normal"
  >
    <p
      className="sc-bdVaJa eAekuO"
      label={false}
      transform="normal"
    >
      default
    </p>
  </styled.p>
</Text>
`;

exports[`should change its style on warn 1`] = `
<Text
  align="left"
  label={false}
  strong={false}
  theme={
    Object {
      "color": Object {
        "black": "#484848",
        "gray": "#f0f0f0",
        "overlay": "rgba(0, 0, 0, .15)",
        "primary": "#0082c8",
        "secondary": "#00baf2",
        "success": "#33bb33",
        "warn": "#dd3333",
        "white": "#fefefe",
      },
      "shape": Object {
        "base": "1rem",
        "border": ".125rem solid rgba(0, 0, 0, .15)",
        "margin": ".25rem .125rem",
        "padding": ".825rem 1.2rem",
        "radius": ".25rem",
        "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
      },
      "text": Object {
        "color": "#484848",
        "size": "1rem",
      },
      "transition": Object {
        "time": ".2s ease",
      },
    }
  }
  transform="normal"
  type=""
  warn={true}
>
  <styled.p
    align="left"
    label={false}
    strong={false}
    theme={
      Object {
        "color": Object {
          "black": "#484848",
          "gray": "#f0f0f0",
          "overlay": "rgba(0, 0, 0, .15)",
          "primary": "#0082c8",
          "secondary": "#00baf2",
          "success": "#33bb33",
          "warn": "#dd3333",
          "white": "#fefefe",
        },
        "shape": Object {
          "base": "1rem",
          "border": ".125rem solid rgba(0, 0, 0, .15)",
          "margin": ".25rem .125rem",
          "padding": ".825rem 1.2rem",
          "radius": ".25rem",
          "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
        },
        "text": Object {
          "color": "#484848",
          "size": "1rem",
        },
        "transition": Object {
          "time": ".2s ease",
        },
      }
    }
    transform="normal"
    warn={true}
  >
    <p
      className="sc-bdVaJa eznnO"
      label={false}
      transform="normal"
    >
      default
    </p>
  </styled.p>
</Text>
`;

exports[`should render a text 1`] = `
<Text
  align="left"
  label={false}
  strong={false}
  theme={
    Object {
      "color": Object {
        "black": "#484848",
        "gray": "#f0f0f0",
        "overlay": "rgba(0, 0, 0, .15)",
        "primary": "#0082c8",
        "secondary": "#00baf2",
        "success": "#33bb33",
        "warn": "#dd3333",
        "white": "#fefefe",
      },
      "shape": Object {
        "base": "1rem",
        "border": ".125rem solid rgba(0, 0, 0, .15)",
        "margin": ".25rem .125rem",
        "padding": ".825rem 1.2rem",
        "radius": ".25rem",
        "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
      },
      "text": Object {
        "color": "#484848",
        "size": "1rem",
      },
      "transition": Object {
        "time": ".2s ease",
      },
    }
  }
  transform="normal"
  type=""
>
  <styled.p
    align="left"
    label={false}
    strong={false}
    theme={
      Object {
        "color": Object {
          "black": "#484848",
          "gray": "#f0f0f0",
          "overlay": "rgba(0, 0, 0, .15)",
          "primary": "#0082c8",
          "secondary": "#00baf2",
          "success": "#33bb33",
          "warn": "#dd3333",
          "white": "#fefefe",
        },
        "shape": Object {
          "base": "1rem",
          "border": ".125rem solid rgba(0, 0, 0, .15)",
          "margin": ".25rem .125rem",
          "padding": ".825rem 1.2rem",
          "radius": ".25rem",
          "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
        },
        "text": Object {
          "color": "#484848",
          "size": "1rem",
        },
        "transition": Object {
          "time": ".2s ease",
        },
      }
    }
    transform="normal"
  >
    <p
      className="sc-bdVaJa jQvpMf"
      label={false}
      transform="normal"
    >
      default
    </p>
  </styled.p>
</Text>
`;
